config.example.json

```diff
@@ -3,5 +3,10 @@
     "password": "yourpassword",
     "min_value": 100,
     "find_add_ons": true,
-    "minutes_between_add_ons_check": 5
+    "minutes_between_add_ons_check": 5,
+    "proxy": [{
```

Comment:
Couldn't `proxy` just be an object instead of an array...

```
"proxy": {
    "server": null,
    "port": null,
    "type": "AUTODETECT"
}
```